src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -394,7 +394,7 @@ public override bool Execute ()
 				? CachePath
 				: Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.LocalApplicationData), CacheBaseDir);
 
-				using (var resolver = new DirectoryAssemblyResolver (LogWarning, loadDebugSymbols: false)) {
+				using (var resolver = new DirectoryAssemblyResolver (Log.CreateTaskLogger (), loadDebugSymbols: false)) {
```

Comment:
@jonpryor not sure we can use it here. Doing that in any Task which uses AsyncTask will lock the UI in VS because we cannot use `Log` on a background thread.  So that code was probably an accident waiting to happen anyway.

Reply:
We should probably have an overload which takes the AsyncTask and uses the LogError and LogWarning methods on that class.